airbyte-webapp/src/packages/cloud/services/ConfigProvider.tsx

```diff
@@ -19,8 +19,8 @@ const configProviders: ValueProvider<Config> = [
   // fileConfigProvider,
   cloudEnvConfigProvider,
   cloudWindowConfigProvider,
-  windowConfigProvider,
```

Comment:
:information_source: Required to make sure `window` has higher priority. This is the same order now than OSS has.